components/add_groups_to_team_modal/add_groups_to_team_modal.tsx

```diff
@@ -162,15 +179,18 @@ export default class AddGroupsToTeamModal extends React.PureComponent {
         }
     }
 
-    handleDelete = (values) => {
+    // public for tests
+    public handleDelete = (values: GroupValue[]): void => {
         this.setState({values});
     }
```

Comment:
```suggestion
    // public for tests
    public handleDelete = (values: GroupValue[]): void => this.setState({values});
```

Reply:
@michelengelen Are there any (Mattermost) code style guidelines that state that is more idiomatic?

Reply:
No, I had a look around and there are non to be found. It is very much my personal style and IMO it simplifies the structure and increases readability. I don't know if there are any implications on performance (I highly doubt that), so it can be either way.

Reply:
I would love for us to unify the different styles used in the code base. Maybe something like https://developers.mattermost.com/contribute/server/style-guide/ could help with that. Are you aware of any eslint plugins that can enforce this style (or any of the other ones you mentioned in this PR)?

Reply:
this is doable by adding this in eslintrc:
```
"rules": {
    "arrow-body-style": ["error", "as-needed"],
 ```
https://eslint.org/docs/rules/arrow-body-style#as-needed
but in our case this rule will not throw error because we don't have `return` keyword in body